src/plugins/score-plugin-avnd/Crousti/ProcessModel.hpp

```diff
@@ -198,8 +198,37 @@ struct InletInitFunc
   void operator()(const T& in, auto idx)
   {
     constexpr auto name = avnd::get_name<T>();
-    auto p = new Process::LineEdit{
-        "", QString::fromUtf8(name.data(), name.size()), Id<Process::Port>(inlet++),
+    auto getFilters = [] <typename P> (P& p)
+    {
+      if constexpr(requires { P::filters().sound; })
+      {
+        QSet<QString> sound_exts = Media::Sound::DropHandler{}.fileExtensions();
+        QString res;
```

Comment:
hmm maybe it should append with a ", ".

You can do: 

    sound_exts.toList().join(", "); 

IIRC

Reply:
Got bamboozled by my ide parser into thinking it wouldn't compile otherwise